README.md

```diff
@@ -153,58 +151,29 @@ To reindex graph manually:
-  </tr>
-  <tr>
-    <td>Code context menu</td>
-    <td>By right-clicking on code, you have an option to render graph for both file and symbol, which will display either full graph per file or all connections for current symbol.</td>
-  </tr>
-  <tr>
-    <td>Reindex menu options</td>
-    <td>By clicking 'Reindex graph (*add mode)' in menu bar (IntelliJ) or in left-side panel (VSCode) you index server with newest data with override or *adding to existing data mode</td>
-  </tr>
-  <tr>
-    <td>Codelens</td>
-    <td>A codelens (tooltip text) that appears above queries in .sql files in .graphbuddy folder, that allows you to run query and display the result on graph / text</td>
-  </tr>
-  <tr>
-    <td>Theming</td>
-    <td>Webview styles change depending on selected IDE theme</td>
-  </tr>
-</table>
+You can do this in multiple ways. The most intuitive is clicking on the code - then corresponding node should appear on plugin canvas.
 
-#### Unique to VSCode
+Second way is using options from context menu in editor or on canvas. In editor there are some options to generate whole graph for selected place in code.
 
-<table>
-  <tr>
-    <td>Side panel</td>
-    <td>A Graph Buddy icon should exist on the left side panel of vscode</td>
-  </tr>
-  <tr>
-    <td>Start extension panel button</td>
-    <td>In plugin panel window a welcome button exists, thats opens up an extension and changes view to advanced options</td>
-  </tr>
-  <tr>
-    <td>Main panel options</td>
-    <td>Panel window contains options such as: reindex options, show graph options, open new Graph Buddy window</td>
-  </tr>
-  <tr>
-    <td>Advanced query editor</td>
-    <td>Window that contains all .sql files and allows to add, delete, rename; files are located in '/workspace/.graphbuddy' folder</td>
-  </tr>
-</table>
+### Filters, layouts, search
+
+You can filter elements of some type using menu on the top of window. Layouts can help you organize nodes on canvas in the way you like. And finally, using search, you can highlight elements on canvas which are matched to typed phrase like `get` or `class`.
+
+### Saving state
+
+If you have created graph, which you want to preserve on the future (f.e. to present it your colleague), you can do this by clicking on floppy disk icon on toolbar. There you can save snaphshot of your current work or load saved previously. You can find those snapshots in `.graphbuddy` folder.
 
-#### Webview-side
+### Regenerating graph files and reindexing
 
-**NOTE: The base network mode for features is 2d, some features are not/less available on 3d graph.**
+It varies a bit depending on language do you use. If you use Java, then after f.e. changing branch some files may be outdated. Currently, this situation is not detected by plugin, so you should click `Generate graph` in such situation.
+If you have a Scala project, then after recompilation you should manually `reindex graph`. It will load newly generated files.
```

Comment:
```suggestion
If you are working with a Scala project, assuming you have configured the compiler plugin correctly, you should manually trigger the `reindex graph` action from the Main Menu. This will load the newly generated graph files to the plugin.
```